.changeset/smart-onions-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/liquid-html-parser': minor
+---
+
+Add parser support for optional liquiddoc parameters
```

Comment:
I know that this isn't the most important thing in the world but might be worth getting in the habit of beefing these up a bit. I know it might not feel like the most important thing now (just like how writing robust commit messages doesn't feel worth it in the moment) but it might be the thing that saves somebody later!

```suggestion
Add support for parsing optional parameters in LiquidDoc. Optional parameters are now designated by wrapping the name in square brackets (e.g. `@param [parameterName]`).
```